setup.py

```diff
@@ -90,7 +90,9 @@
 extras["all"] = extras["serving"] + ["tensorflow", "torch"]
 
 extras["retrieval"] = ["faiss-cpu", "datasets"]
-extras["testing"] = ["pytest", "pytest-xdist", "timeout-decorator", "parameterized", "psutil"] + extras["retrieval"]
+extras["testing"] = ["pytest", "pytest-xdist", "timeout-decorator", "parameterized", "psutil", "comet_ml"] + extras[
```

Comment:
I'm not fond of adding integrations to the "testing" extra dependency as they are already quite a few of them (between logging and hyperparameter search) and it would then slow down our CI quite a bit to install everything (on every commit).

We can add a "testing-integrations" flag if we want to add some tests that we run on a slower schedule (like once a day).

Reply:
Totally agree! I think we'll create a new repo for testing, and run often so we can catch errors likes this. Shall I remove the test commits, or do you want to cherry pick?

Reply:
If you can remove them it's easier for me to merge :-)

Reply:
No problem, will do.

Reply:
All done! (since I touched setup.py, black insists on reformatting it. Not sure how to get it back to original state without re-running black on it, without creating a new PR.)

Reply:
As long as the CI is happy, I'm good with the change ;-) Thanks!